docs/AndroidBuildingFromSource.md

```diff
@@ -20,13 +20,25 @@ Make sure you have the following installed:
 3. Local Maven repository for Support Libraries (formerly `Android Support Repository`) >= 17 (for Android Support Library)
 4. Android NDK (download links and installation instructions below)
 
-Point Gradle to your Android SDK: either have `$ANDROID_SDK` and `$ANDROID_NDK ` defined, or create a local.properties file in the root of your react-native checkout with the following contents:
+### Point Gradle to your Android SDK:
+
+**Step 1:**  Set environment variables through your local shell.
```

Comment:
Isn't it more option 1 / option 2, instead of steps?
I didn't set any environment variable, I just created `android/local.properties`